code/lib/cli/src/automigrate/fixes/mdx-gfm.ts

```diff
@@ -22,9 +25,27 @@ export const mdxgfm: Fix<Options> = {
       return null;
     }
 
+    const hasMDXFiles = await mainConfig?.stories?.reduce(async (acc, item) => {
```

Comment:
I wonder if you could just do the same as this automigration: https://github.com/storybookjs/storybook/blob/next/code/lib/cli/src/automigrate/fixes/mdx-1-to-2.ts#L49

Reply:
That seems worse? considering it's not using the glob patterns set in `main.ts`, and will ignore any MDX files not following the `.stories.mdx` pattern.

Reply:
Alright sounds good, maybe the other automigration should follow this pattern then